CHANGELOG.md

```diff
@@ -4,6 +4,21 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.0.0 (2020-12-17)
+
+This is a major release concluding several release candidates and it delivers
+several major features to GA:
+* #1923 Add support for distributed hypertables
+* #2006 Add support for user-defined actions
+* #2435 Move enterprise features to community
+* #2437 Update Timescale License
```

Comment:
Do we want to mention the following here as well, since they are significant changes:

- Rewritten continuous aggregates
- Refactored informational views
- Improved memory profile for COPY

Reply:
I think it is good to mention the first two. I don't think improvements in COPY is a major feature.